### ISSUE TITLE
Issues reading mzXML files

### ISSUE THREAD

**Issue by saersamani** [timholy/MzXML.jl#1]
Hi, 

I am new to Julia and tried to read an mzXML file using your package and ran the following script:

out=mzXML.load(path)
out1=mzXML.index(path)

Running this code I get the following error. 

UndefVarError: readbytes not defined
loadrest(::IOStream) at mzXML.jl:143
(::mzXML.##1#2)(::IOStream) at mzXML.jl:31
open(::mzXML.##1#2, ::String) at iostream.jl:152
index(::String) at mzXML.jl:22
include_string(::String, ::String) at loading.jl:522
include_string(::String, ::String, ::Int64) at eval.jl:30
include_string(::Module, ::String, ::String, ::Int64, ::Vararg{Int64,N} where N) at eval.jl:34
(::Atom.##49#53{String,Int64,String})() at eval.jl:50
withpath(::Atom.##49#53{String,Int64,String}, ::Void) at utils.jl:30
withpath(::Function, ::String) at eval.jl:38
macro expansion at eval.jl:49 [inlined]
(::Atom.##48#52{Dict{String,Any}})() at task.jl:80

Could you please help me to solve this issue.
Cheers,
Saer

**Comment by timholy** [comment]
I think this package was written a long time ago and sat on my harddrive until I eventually pushed it to GitHub. `readbytes` appears to have been a 0.4 function. In julia-0.5 you get this:
```julia
julia> readbytes(io)
WARNING: readbytes is deprecated, use read instead.
 in depwarn(::String, ::Symbol) at ./deprecated.jl:64
 in readbytes(::Base.AbstractIOBuffer{Array{UInt8,1}}, ::Vararg{Base.AbstractIOBuffer{Array{UInt8,1}},N}) at ./deprecated.jl:30
 in eval(::Module, ::Any) at ./boot.jl:234
 in eval_user_input(::Any, ::Base.REPL.REPLBackend) at ./REPL.jl:64
 in macro expansion at ./REPL.jl:95 [inlined]
 in (::Base.REPL.##3#4{Base.REPL.REPLBackend})() at ./event.jl:68
while loading no file, in expression starting on line 0
5-element Array{UInt8,1}:
 0x48
 0x65
 0x6c
 0x6c
 0x6f
```
Deprecation warnings are emitted only for one release cycle, after which they become errors.

So what you might need to do is install julia-0.5, try running the package and fix any deprecation warnings you get. Then take the fixed code and run it in julia-0.6, again fixing any deprecation warnings you get. Then, please submit a pull request!

**Comment by timholy** [comment]
Hmm, if you're new to Julia then this will be hard for you. See #2.
